RELEASING.md

```diff
@@ -6,7 +6,9 @@ Releasing notes
 - dry run:
   - `sbt -Dsbt.supershell=false clean ^plugin/publishLocalSigned )`
 - release the plugin:
+  -  prerequisite: @ktoso needs to grant your oss.sontaype.org publish permissions by means of a request to [OSSRH-1324](https://issues.sonatype.org/browse/OSSRH-1324).
```

Comment:
Alternatively we could perhaps just change the groupId of the plugin to one we have publish permissions for already?

Reply:
You and I both have permissions for the current group ID. What alternative did you have in mind?

Reply:
Ah, we do, I had forgotten that and thought this was a reminder for us. Everything good then!